monkey/monkey_island/cc/ui/src/components/IslandHttpClient.tsx

```diff
@@ -21,10 +21,38 @@ export enum APIEndpoint {
   stolenCredentials = '/api/propagation-credentials/stolen-credentials',
   linuxMasque = '/api/agent-binaries/linux/masque',
   windowsMasque = '/api/agent-binaries/windows/masque',
-  installAgentPlugin = '/api/install-agent-plugin'
+  installAgentPlugin = '/api/install-agent-plugin',
+  uninstallAgentPlugin = 'api/uninstall-agent-plugin'
 }
 
 class IslandHttpClient extends AuthComponent {
```

Comment:
This whole class needs to be refactored, but for now at least remove the duplication between `_put` and `_post`